invenio_communities/assets/semantic-ui/js/invenio_communities/details_search/components.js

```diff
@@ -124,11 +134,44 @@ export const CommunityRecordResultsGridItem = ({ result }) => {
   );
 };
 
-export const CommunityRecordSearchBarContainer = () => {
+export const CommunityRecordSearchAppLayout = (props) => {
+  const [sidebarVisible, setSidebarVisible] = React.useState(false);
+
   return (
-    <Overridable id={"SearchApp.searchbar"}>
-      <SearchBar />
-    </Overridable>
+    <Container>
+      <div className="rel-pt-2">
+        <Grid>
+          <Grid.Column only="mobile tablet" mobile={2} tablet={1}>
+            <Button
+              basic
+              icon="sliders"
+              onClick={() => setSidebarVisible(true)}
+              aria-label={i18next.t("Filter results")}
+            />
+          </Grid.Column>
+
+          <Grid.Column mobile={16} tablet={15} computer={12} floated="right">
+            <SearchBar
+              placeholder={i18next.t("Search records in community...")}
+            />
+          </Grid.Column>
+
+          <Grid.Row>
+            <GridResponsiveSidebarColumn
+              width={4}
+              open={sidebarVisible}
+              onHideClick={() => setSidebarVisible(false)}
```

Comment:
IMO the sidebar should manage its own state (visible/hidden), not its parent

Reply:
I think that'll be tricky as the state is connected to the filter-button (line 145)?

Reply:
The state is lifted because the sidebar is externally controlled...